platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/IdpEndpoint.java

```diff
@@ -146,17 +166,19 @@
 import org.w3c.dom.Node;
 
 @Path("/")
-public class IdpEndpoint implements Idp {
+public class IdpEndpoint implements Idp, SessionHandler {
 
   public static final String SERVICES_IDP_PATH = SystemBaseUrl.getRootContext() + "/idp";
+  public static final ImmutableSet<UsageType> USAGE_TYPES =
+      ImmutableSet.of(UsageType.UNSPECIFIED, UsageType.SIGNING);
```

Comment:
It may not be (definitely isn't within DDF itself), but as it's public and may be used by a downstream project, I'm loath to remove it as part of this PR.